src/komponenter/ditt-sykefravaer/ditt-sykefravaer.test.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+import { render, screen } from '@testing-library/react';
+import { contextProviders } from '../../test/test-context-providers';
+import tekster from '../../tekster/tekster';
+import DittSykefravaer from './ditt-sykefravaer';
+
+describe('Tester at komponenten rendres som den skal', () => {
+    test('Komponenten vises IKKE dersom ikke sykefravær', () => {
+        const providerProps = {};
+        const { container } = render(<DittSykefravaer />, { wrapper: contextProviders(providerProps) });
+        expect(container).toBeEmptyDOMElement();
+    });
+
+    test('Komponenten VISES dersom ikke erSykmeldtMedArbeidsgiver', async () => {
+        const providerProps = {
```

Comment:
I går oppdaget jeg noen rare feil ved opprydding, som viste seg å være at `providerProps` inneholdt ugyldige verdier, og ble dermed bare forkastet i `contextProviders`. For å fikse opp i dette, la jeg til en type på objektet, og da fikk jeg tilbakemeldinger fra kompilatoren om hva som var galt med en gang, og kunne rette opp i det. Med en type får jeg også autocomplete på hvilke props jeg kan sette her :)
altså:
```typescript
const providerProps: ProviderProps = {
   ...
}
```
Ingen krise akkurat, men jeg prøver å alltid legge det til for sikkerheten det gir

Reply:
kan legge det til... men får slike varsler av VS code by default fra før altså :-)